### PR TITLE
Polish development environment for Message Broker

### PR DESCRIPTION
**Pull request by amiedes**
Related: https://app.clubhouse.io/cartoteam/story/119784/wait-for-the-pubsub-emulator-to-be-ready-before-provisioning

This is a part of 3 related PR's:

- https://github.com/CartoDB/cartodb/pull/15982
- https://github.com/CartoDB/cartodb-central/pull/2994
- https://github.com/CartoDB/docker-dev-env/pull/72

It's worth saying that this PR can be merged independently from the other two.

### CODE REVIEW

File: lib/tasks/central_updates_subscriber.rake [hunk 1]
```diff
@@ -1,16 +1,35 @@
...
+
+  begin
+    ActiveRecord::Base.connection
+  rescue ActiveRecord::NoDatabaseError => e
+    raise e if retries >= max_retries
+
+    retries += 1
+    database_name = Rails.configuration.database_configuration[Rails.env]['database']
+    logger.info("Database '#{database_name}' doesn't exist. Sleeping 5 seconds before retry #{retries}/#{max_retries}")
+    sleep(5)
```
Comment (rafatower) [inline lib/tasks/central_updates_subscriber.rake:15]: As a convenience, I'd lower the sleep time to 1s and raise the max_retries to 25x5=125

File: lib/tasks/central_updates_subscriber.rake [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
     pid_file = ENV['PIDFILE'] || Rails.root.join('tmp/pids/cartodb_subscribers.pid')
-    raise 'pid file exists!' if File.exist?(pid_file)
+    raise "PID file exists: #{pid_file}" if File.exist?(pid_file)
 
     File.open(pid_file, 'w') { |f| f.puts Process.pid }
     begin
       $stdout.sync = true
       logger = Carto::Common::Logger.new($stdout)
 
+      wait_for_db_creation(logger) if Rails.env.development? || Rails.env.test?
```
Comment (rafatower) [inline lib/tasks/central_updates_subscriber.rake:31]: > Rails.env.test?

We're not using it in tests AFAIK (and hope not to have to use it in automated tests - demonstrating a real "loose coupling" at least between central-broker-cartodb), but OK.

If you have no strong opinion against, I'd delete it.

**Review by rafatower — APPROVED** [review]

**Comment by rafatower** [comment]
PS: consider moving the `wait_for_db_creation` functionality to the caller script, instead of the subscriber rake.

**Comment by amiedes** [comment]
Changes addressed!

**Pull request merged**